tests/e2e/specs/modules/adsense/amp-auto-ads-tag.test.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * AMP auto ad tag tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import {
+	deactivatePlugin,
+	activatePlugin,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	setAMPMode,
+	setupAdSense,
+} from '../../../utils';
+
+describe( 'AMP <amp-auto-ads> tag', () => {
+	beforeAll( async () => {
+		await setupAdSense();
+		await activatePlugin( 'amp' );
+		await activatePlugin( 'e2e-tests-apply-content-filters' );
+	} );
+	afterAll( async () => {
+		await deactivatePlugin( 'amp' );
+		await deactivatePlugin( 'e2e-tests-apply-content-filters' );
+	} );
+
+	it( 'is output in primary mode', async () => {
+		await setAMPMode( 'primary' );
+		await expect( '/hello-world' ).toHaveAMPAutoAdsTag();
+	} );
+
+	it( 'is output in secondary mode', async () => {
+		await setAMPMode( 'secondary' );
+		await expect( '/hello-world' ).toHaveAMPAutoAdsTag();
+	} );
```

Comment:
As previously mentioned, let's also test here that the auto ads tag is _not_ output on the web version (not `?amp`) when in secondary AMP mode. 

For some reason it won't let me suggest on this file, but how about this for clarity?
* `is output in secondary mode in AMP context` _(existing test)_
* `is not output in secondary mode in web context` _(test to add)_